portfolio/src/main/webapp/script.js

```diff
@@ -442,15 +456,72 @@ function displayDirectionsAPI() {
   }
 }
 
-/*
-function repeatMarkers() {
-    fetch('/markers').then(response => response.json()).then((markers) => {
-    markers.forEach((marker) => {
-        var myPosition = new google.maps.LatLng(marker.lat, marker.lng);
-        if (google.maps.geometry.poly.isLocationOnEdge(myPosition, routeArray, 0.0064)) {
-            
-        }
-        });
+function repeatedMarkersUI() {
+    console.log("HEYYYYY");
+  document.getElementById('repeatedMarkerUI').style.display = "block";
+  setTimeout(hidePopup, 3000);
+}
+
+function hidePopup() {
```

Comment:
bad name. There's going to be more than one popup and this doesn't even seem to be a popup. rename this to hideRepeatMarkerPopup or something